tests_results/assets/style.css

```diff
@@ -0,0 +1,186 @@
+body {
```

Comment:
Should the test_results directory be removed from this PR?

Reply:
ooh yes I'll remove it.. I thought I added this to the `.gitignore`